ArduPlane/ArduPlane.cpp

```diff
@@ -341,6 +345,35 @@ void Plane::one_second_loop()
     // indicates that the sensor or subsystem is present but not
     // functioning correctly
     update_sensor_status_flags();
+
+
+#if HAL_WITH_UAVCAN
```

Comment:
you could move this sort of debug into AP_UAVCAN, triggered when the DEBUG parameter for the can port is 4 or higher (or some similar trigger level) and tunnelling is enabled

Reply:
true, if sending GCS messages from AP_UAVCAN.cpp worked. I haven't tracked down that problem (yet).